namespaces?limit=1

```diff
@@ -0,0 +1 @@
+[{"name":"default"}]
```

Comment:
Is this intentional?

Reply:
![](https://lesjoiesducode.fr/content/041/CasKTWn.gif)